src/InvalidArgumentException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace JakubOnderka\PhpParallelLint;
+
+class InvalidArgumentException extends Exception
```

Comment:
same, avoid using same class names as present in root namespace.

Reply:
This feedback belongs with PR #44.